modules/ding_nodelist/ding_nodelist.install

```diff
@@ -184,6 +124,9 @@ function ding_nodelist_install() {
     );
     db_insert('ding_nodelist_templates')->fields($fields)->execute();
   }
+
+  // Create new "colored" css on module install.
+  _ding_nodelist_process_css_colors();
```

Comment:
@NicolaeGrigoriu Not sure if this functionality should be kept for ding2/diing2 as we have styling for ding_nodelist in ddbasic theme and stylesheets which are processed automatically.
@guddo Should the still keep this here?